src/js/vic-v2/components/VeteranIDCard.jsx

```diff
@@ -1,7 +1,34 @@
 import React from 'react';
 
+// will be able to set either height or width of component but not both:w
+const WIDTH_HEIGHT_RATIO = 1.583;
+const height = 300;
+
 export default function VeteranIDCard() {
+  // <img className="example-photo" alt="placeholder" src="/img/air-force-coat-of-arms.png"/>
   return (
-    <div style={{ paddingTop: '95px', backgroundColor: 'gray', height: '200px', width: '300px' }}></div>
+    <div style={{ marginTop: '50px', backgroundColor: 'grey', height, width: height * WIDTH_HEIGHT_RATIO }}>
+      <svg width={'100%'} height={'100%'}>
+        <image width={'100%'} href={'/img/vic-preview-background.svg'} />
```

Comment:
Is the number of branches/discharge statuses small enough that it could be worthwhile to pre-generate all possible versions of the card image so that we only need to specify the name, id, and profile photo on the appropriate pre-generated card background?

Reply:
That would allow the removal 3 text tags and 2 image tags from our code.